tests/draft3/ref.json

```diff
@@ -141,6 +141,32 @@
         ]
     },
     {
+        "description": "ref, containing other keywords",
```

Comment:
Would be a bit clearer I think to say something like "ref overrides any sibling keywords" which is the behavior, rather than just describing the situation without explicitly pointing that out.